File: BitcoinCheckerDataModule/src/com/mobnetic/coinguardian/model/Ticker.java
package com.mobnetic.coinguardian.model;


public class Ticker {

	public double bid;
	public double ask;
	public double vol;
	public double high;
	public double low;
	public double last;
	public long timestamp;
	
	public final static int NO_DATA = -1;
	
	public Ticker() {
		bid = NO_DATA;
		ask = NO_DATA;
		vol = NO_DATA;
		high = NO_DATA;
		low = NO_DATA;
		last = NO_DATA;
		timestamp = NO_DATA;
	}
}

File: dataModule/src/main/java/com/aneonex/bitcoinchecker/datamodule/model/Ticker.kt
package com.aneonex.bitcoinchecker.datamodule.model

interface Ticker {
	var bid: Double
	var ask: Double
	var vol: Double
    var volQuote: Double
	var high: Double
	var low: Double
	var last: Double
	var timestamp: Long

	companion object {
		const val NO_DATA = -1
	}
}